k8s/server/mosquitto.yaml

```
@@ -18,17 +18,15 @@ spec:
         app.kubernetes.io/instance: mosquitto
         app.kubernetes.io/name: mosquitto
     spec:
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-            - matchExpressions:
-              - key: mqhost
-                operator: In
-                values:
-                - "true"
       containers:
       - image: eclipse-mosquitto:2.0.11-openssl
+        command: ["/bin/ash", "-c"]
+        args: ["echo \"REPLACE_MQ_USERNAME:REPLACE_MQ_PASSWORD\" > /mosquitto/password.txt && mosquitto_passwd -U /mosquitto/password.txt && /docker-entrypoint.sh && /usr/sbin/mosquitto -c /mosquitto/config/mosquitto.conf"]
```

```suggestion
        args: ["echo \"$MQ_USERNAME:$MQ_PASSWORD\" > /mosquitto/password.txt && mosquitto_passwd -U /mosquitto/password.txt && /docker-entrypoint.sh && /usr/sbin/mosquitto -c /mosquitto/config/mosquitto.conf"]
```

Wouldn't this work cleaner?